airflow/providers/amazon/provider.yaml

```diff
@@ -828,6 +828,19 @@ config:
         type: string
         example: "ecs_executor_cluster"
         default: ~
+      capacity_provider_strategy:
+        description: |
+          The capacity provider strategy to use for the task.
+
+          If a capacityProviderStrategy is specified, the launchType parameter must be omitted. If
+          no capacityProviderStrategy or launchType is specified, the defaultCapacityProviderStrategy
+          for the cluster is used, if present.
+
+          When you use cluster auto scaling, you must specify capacityProviderStrategy and not launchType.
+        version_added: "8.13"
```

Comment:
next release is 8.17

Reply:
Gah, sorry, thanks.  This one sat on the back burner for a while (clearly).  I'll update that.